include/boost/geometry/formulas/differential_quantities.hpp

```diff
@@ -73,6 +74,9 @@ class differential_quantities
             CT const sig_12 = dlon / one_minus_f;
             if (BOOST_GEOMETRY_CONDITION(EnableReducedLength))
             {
+                CT const pi = math::pi<CT>();
+                BOOST_GEOMETRY_ASSERT(-pi <= azimuth && azimuth <= pi);
```

Comment:
You will get a warning for an unused variable pi in release mode. Maybe just use the math::pi<CT>() inside the assertion?